src/concepts/components/properties.md

```diff
@@ -1,26 +1,551 @@
 ---
-description: Parent to child communication
+description: The component could expose property attributes to receive data from the parent html component
 ---
 
 # Properties
 
-Properties enable child and parent components to communicate with each other.
+It is a common need to receive data from the parent component using properties.
```

Comment:
I don't think it should say "it is a common need... using properties" but rather something along the lines of "as an application becomes more complex, it becomes necessary to split it up into many different components to achieve an effective separation of concerns. For these components to be able to communicate with each other, components have properties – these are values which their parent components can pass to them."

Reply:
Not sure it's necessary to have "as an application becomes more complex, it becomes necessary to split it up into many different components to achieve an effective separation of concerns." 
In some way it should be the introduction of Components.

Anyway, I paste your whole proposal.